test/AutoRest.TestServer.Tests/constantsClient.cs

```diff
@@ -42,8 +42,6 @@ public void PutNoModelAsStringNoRequiredTwoValueDefault_HasOptionalDefaultParam(
             TypeAsserts.HasParameter(method, "cancellationToken");
             var paramInfo = TypeAsserts.HasParameter(method, "input");
             Assert.AreEqual(typeof(NoModelAsStringNoRequiredTwoValueDefaultOpEnum?), paramInfo.ParameterType);
-            Assert.True(paramInfo.HasDefaultValue);
-            Assert.AreEqual((int)NoModelAsStringNoRequiredTwoValueDefaultOpEnum.Value1, paramInfo.DefaultValue);
```

Comment:
Instead of removing these, could we validating `paramInfo.HasDefaultValue` is false here?
Also please add some comments here to describe we explicitly ignored the client default value.